### PR TITLE
Add card

### PR DESCRIPTION
**Pull request by Tombean**

### CODE REVIEW

**Comment by TheoGauchoux** [comment]
Can you enlarge the button to match the cards' width ?

screen shot 2017-10-16 at 19 36 00

File: src/components/card/style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.card{
```
Comment (TheoGauchoux) [inline src/components/card/style.css]: Space required before the `{ `

File: src/components/cards/Cards.jsx [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
 const Cards = props => (
-  <div>
+  <div className="cardsWrapper">
     {props.cards.map(card => (
       <Card key={card.id} {...card} />
     ))}
+    <div className="addCardBlock">
+      <Button
+        className="addCardButton"
+        // TODO : fix listID call in route.
```
Comment (TheoGauchoux) [inline src/components/cards/Cards.jsx]: Remove this useless comment.

File: src/components/cards/style.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.cardsWrapper {
```
Comment (TheoGauchoux) [inline src/components/cards/style.css:1]: Order the properties by alphabetical order

File: src/components/cards/style.css [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+.addCardBlock {
+    display: inline-flex;
+    margin: 13px 10px 0 10px;;
+}
+
+.addCardButton {
+    display: inline-flex;
+    width: 250px;
```
Comment (TheoGauchoux) [inline src/components/cards/style.css]: The button's width needs to be the same as a card.

File: src/components/cards/style.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    padding: 0 20px 0 20px;
+    overflow-y: auto;
+}
+
+.addCardBlock {
+    display: inline-flex;
+    margin: 13px 10px 0 10px;;
+}
+
+.addCardButton {
```
Comment (TheoGauchoux) [inline src/components/cards/style.css]: We need to order those properties by alphabetical order

File: src/components/cards/style.css [hunk 6]
```diff
@@ -0,0 +1,20 @@
+.cardsWrapper {
+    display: flex;
+    flex-direction: column;
+    height: 70vh ;
```
Comment (TheoGauchoux) [inline src/components/cards/style.css]: Remove the space

File: src/components/cards/style.css [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    width: 100%;
+    padding: 0 20px 0 20px;
+    overflow-y: auto;
+}
+
+.addCardBlock {
+    display: inline-flex;
+    margin: 13px 10px 0 10px;;
```
Comment (TheoGauchoux) [inline src/components/cards/style.css]: Remove the double `;`

File: src/components/list/style.css [hunk 8]
```diff
@@ -1,9 +1,14 @@
 .list {
     display: inline-grid;
     margin: 10px;
-    width: 300px;
+    width: 370px;
+    /*overflow-x: hidden;*/
```
Comment (TheoGauchoux) [inline src/components/list/style.css]: Remove this unused property

File: src/components/list/style.css [hunk 9]
```diff
@@ -1,9 +1,14 @@
...
 }
 
 .draggableList {
     display: inline-grid;
+}
+
+.list > .ant-card-body{
+    padding: 0!important;
```
Comment (TheoGauchoux) [inline src/components/list/style.css]: You could remove the `!important`

File: src/components/list/style.css [hunk 10]
```diff
@@ -1,9 +1,14 @@
...
-    width: 300px;
+    width: 370px;
+    /*overflow-x: hidden;*/
 }
 
 .draggableList {
     display: inline-grid;
+}
+
+.list > .ant-card-body{
```
Comment (TheoGauchoux) [inline src/components/list/style.css]: Add a space before the `{ `

File: src/components/lists/style.css [hunk 11]
```diff
@@ -1,8 +1,8 @@
-.listWrapper {
+.listsWrapper {
     display: inline-flex;
     height: inherit;
     overflow-x: scroll;
-    padding: 0 50px 0 10px;
+    margin: 0 0px 0 0px;
```
Comment (TheoGauchoux) [inline src/components/lists/style.css]: You can write `margin: 0;`

File: src/containers/cards/actions.js [hunk 12]
```diff
@@ -1,4 +1,21 @@
-import { GET_ALL_CARDS_IN_LIST } from './constants'
+import { ADD_CARD, GET_ALL_CARDS_IN_LIST } from './constants'
+
+export const addCard = listId => (dispatch) => {
+  console.log(`List ID : ${listId}`)
```
Comment (TheoGauchoux) [inline src/containers/cards/actions.js]: Remove the `console.log`

File: src/containers/cards/constants.js [hunk 13]
```diff
@@ -1 +1,4 @@
+export const ADD_CARD = 'lists/ADD_CARD'
 export const GET_ALL_CARDS_IN_LIST = 'cards/GET_ALL_CARDS_IN_LIST'
+export const DELETE_CARD = 'cards/DELETE_CARD'
```
Comment (TheoGauchoux) [inline src/containers/cards/constants.js]: Why do you have a `DELETE_CARD` and `SAVE_CARD_RANK` in this PR ? It's useless.

**Review by TheoGauchoux — CHANGES_REQUESTED** [review]

**Comment by TheoGauchoux** [comment]
Some errors of inattention, but the logic is good ! GG 👍

File: src/components/card/style.css (hunk 1 above)
Comment (Tombean) [reply src/components/card/style.css]: Done

File: src/components/cards/Cards.jsx (hunk 2 above)
Comment (Tombean) [reply src/components/cards/Cards.jsx]: Done

File: src/components/cards/style.css (hunk 3 above)
Comment (Tombean) [reply src/components/cards/style.css:1]: Done

File: src/components/cards/style.css (hunk 6 above)
Comment (Tombean) [reply src/components/cards/style.css]: Done

File: src/components/cards/style.css (hunk 7 above)
Comment (Tombean) [reply src/components/cards/style.css]: Done

File: src/components/cards/style.css (hunk 5 above)
Comment (Tombean) [reply src/components/cards/style.css]: Done

File: src/components/cards/style.css (hunk 4 above)
Comment (Tombean) [reply src/components/cards/style.css]: Done

File: src/components/list/style.css (hunk 8 above)
Comment (Tombean) [reply src/components/list/style.css]: Done

File: src/components/list/style.css (hunk 10 above)
Comment (Tombean) [reply src/components/list/style.css]: Done

File: src/components/list/style.css (hunk 9 above)
Comment (Tombean) [reply src/components/list/style.css]: Done

File: src/components/lists/style.css (hunk 11 above)
Comment (Tombean) [reply src/components/lists/style.css]: Done

File: src/containers/cards/actions.js (hunk 12 above)
Comment (Tombean) [reply src/containers/cards/actions.js]: Done

File: src/containers/cards/constants.js (hunk 13 above)
Comment (Tombean) [reply src/containers/cards/constants.js]: Done

**Review by TheoGauchoux — APPROVED** [review]

**Pull request merged**